_inc/lib/tracks/class.tracks-event.php

```diff
@@ -0,0 +1,5 @@
+<?php
+/**
+ * Deprecated since 7.5 – Tracks_Event are autoloaded from REPLACEMENT
+ */
+_deprecated_file( basename( __FILE__ ), 'jetpack-7.5', 'ADD REPLACEMENT_PATH' );
```

Comment:
For all of these: Did you intent to leave `ADD REPLACEMENT_PATH` as the replacement? This will be echo'd in the notice on the site.

Reply:
If it's a pain to get the changed path, then I'd be ok with just omitting that parameter and be done with it. If we want, there is a 4th parameter that we can add our own global custom message that is more generic and can be scripted.